lib/ood_core/job/adapters/kubernetes/templates/pod.yml.erb

```diff
@@ -52,6 +52,10 @@ spec:
       valueFrom:
         fieldRef:
           fieldPath: metadata.namespace
+    <%- unless script.workdir.nil? -%>
+    - name: WORKDIR
+      value: "<%= script.workdir %>"
+    <%- end -%>
```

Comment:
Why is `WORKDIR` needed, isn't this just `$PWD`?

Reply:
This is only way I could find to get OnDemand staged_root into the pod.  It's not technically needed since I think OnDemand job script will do like `cd <staged_root>` but if that's not always the case then I figured a command like this would be used:

```
cd $(WORKDIR) ; /ood/script.sh
```

It's also the case this is not needed if the script templates all do the `cd <%= staged_root %>`.  Unfortunately to make `PWD` be staged root, would require updating the code in pod.yml.erb to also allow setting working directory to path defined in `script.workdir`.  If Ondemand will always do `cd staged_root` then this environment variable isn't needed.

Reply:
I removed `WORKDIR` env variable from this pull request. If we need it in the future it's easy enough to add back. This pull request now just focuses on making job script accessible to pod.